test/deadline_submitter_for_houdini/unit/test_job_template.py

```diff
@@ -0,0 +1,184 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+from unittest.mock import Mock, patch
+import pytest
+from deadline.houdini_submitter.python.deadline_cloud_for_houdini.submitter import (
+    _get_job_template,
+    RenderStrategy,
+)
+
+
+@pytest.fixture
+def mock_parm():
+    def mock_implementation(name: str):
+        if name == "name":
+            mock_name = Mock()
+            mock_name.evalAsString.return_value = "My Job"
+            return mock_name
+        if name == "include_adaptor_wheels":
+            mock_name = Mock()
+            mock_name.eval.return_value = False
+            return mock_name
+        if name == "description":
+            mock_name = Mock()
+            mock_name.evalAsString.return_value = "My job description"
+            return mock_name
+        return Mock()
+
+    return mock_implementation
+
+
+@pytest.fixture(autouse=True)
+def init_mocks():
+    with patch(
+        "deadline.houdini_submitter.python.deadline_cloud_for_houdini.submitter._get_houdini_version",
+        Mock(return_value="19.5.435"),
+    ):
+        with patch(
+            "deadline.houdini_submitter.python.deadline_cloud_for_houdini.submitter._get_hip_file",
+            Mock(return_value="/path/to/hip.hip"),
+        ):
+            yield
+
+
+@patch("deadline.houdini_submitter.python.deadline_cloud_for_houdini.submitter._get_steps")
```

Comment:
nit: `_get_steps` is also mocked twice in this file, can it be moved into a fixture?